lib/src/worker/worker_connection.dart

```diff
@@ -57,6 +65,35 @@ class StdAsyncWorkerConnection implements AsyncWorkerConnection {
   }
 }
 
+/// Implementation of [AsyncWorkerConnection] for running in an isolate.
+class SendPortAsyncWorkerConnection implements AsyncWorkerConnection {
+  final ReceivePort receivePort;
+  final StreamIterator receivePortIterator;
```

Comment:
nit: cast this as a `StreamIterator<Uint8List>` instead of manually casting each item we pull out (wont be less work but gives us a better type for the public field).

Reply:
Done.